### ISSUE TITLE
[FEATURE] Auto fix T002, T003,  but potentially any auto-fixable issue

### ISSUE THREAD

**Issue by x-xyzxyz** [djlint/djLint#475]
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## Feature Request

Perhaps similar to https://github.com/Riverside-Healthcare/djLint/issues/252, so I'm linking it for cross-reference.

I was wondering if there's a way to enable auto fixing of certain problems like for example the two mentioned below?

It does work for some problems _(like adding spaces inside a variable)_, but it'd be really nice if it could work with these too.

## Example

Given this code:

```twig
{% block name %}
   {% include 'header.twig' %}
{% endblock %}
```

These problems are reported, based on my .djlintrc config:
1. ```Endblock should have name. Ex: {% endblock body %}. {% endblock %} (T003)```
2. ```Double quotes should be used in tags. {% include 'header.t (T002)```

And ideally the code above should be replaced with the following on save:
```twig
{% block name %}
   {% include "header.twig" %}
              ^           ^
{% endblock name %}
            ^^^^
```

Thank you :)

**Comment by rennerocha** [comment]
This is what I would expect for an automatic formatter. `black` for example, fixed the quotes of a file automatically.

**Comment by monosans** [comment]
Both are auto-fixable now.

T002 already lands via `--reformat`. The formatter rewrites tag arguments to the configured `--quote-style` (`double` by default), so `{% include 'header.twig' %}` becomes `{% include "header.twig" %}`.

T003 is new in v1.45.0 as `--name-endblocks` / `name_endblocks`. Your example:

```console
$ djlint page.twig --reformat --name-endblocks
```

```twig
{% block name %}
    {% include "header.twig" %}
{% endblock name %}
```

`djlint page.twig --lint --include T003` is then clean.

It writes the name only where `T003` asks for one. A block opened and closed on the same line already says which block it closes, so `{% block a %}x{% endblock %}` is left alone. A block shown as text inside `{% raw %}`, `{% verbatim %}` or `<pre>` opens nothing, and a block with no name has none to copy.

It is off by default, matching `T003` itself.
